tracer/test/Datadog.Trace.Tools.Runner.ArtifactTests/LegacyCommandLineArgumentsTests.cs

```diff
@@ -73,6 +71,20 @@ public void SetCi(char separator)
             environmentVariables["VAR2"].Should().Be("B");
         }
 
+        [Fact]
+        public void LocateTracerHome()
+        {
+            var commandLine = "--set-ci";
+
+            using var helper = StartProcess(commandLine, ("TF_BUILD", "1"));
+
+            helper.Process.WaitForExit();
+            helper.Drain();
+
+            helper.Process.ExitCode.Should().Be(0);
+            helper.StandardOutput.Should().NotContainEquivalentOf("error");
```

Comment:
Haven't dug into it, but do we collect StandardError here too? If so, should we check it for empty?

Reply:
I don't think we write to stderr even when it fails, but it doesn't hurt to check